src/libraries/System.Net.Quic/src/System/Net/Quic/QuicConnection.cs

```diff
@@ -13,6 +13,8 @@
 using Microsoft.Quic;
 using static Microsoft.Quic.MsQuic;
 
+// test
+
```

Comment:
not was intentional.... somewhat It seems like different piplenes run depending on what changed. This (and some other intermediate changes) is attempt to get desired run without running `extra-platforms` because that is is long and pretty expensive. 
I end up doing it anyway ;( and this was always meant to be remove before merging. 
I probably should have put in some comments explaining.